Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -312,8 +315,13 @@ else if (notificationDeepLinkVector != null) {
         initApp(viewParam, deepLinkJson);
 
         mFirebaseAnalytics.logEvent(isMigrated ?"migrate_local_store_success" : "migrate_local_store_failed",new Bundle());
+        initNotificationChannel();
         CleverTapAPI cleverTap = CleverTapAPI.getDefaultInstance(context);
-        CleverTapAPI.createNotificationChannel(context,clientId,clientId,"notification",NotificationManager.IMPORTANCE_MAX,true);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
x-variable = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O ? NotificationManager.IMPORTANT_MAX : NotificationManager.IMPORTANCE_MAX;

CleverTapAPI.createNotificationChannel(context,clientId,"Promotion","Notifications Related to promotion",x-variable,true);

Reply:
this looks cleaner

Reply:
i checked that condintion for groupId parameter in `createNotificationChannel`. For that, we can't write like this.

Reply:
because in older version there is no group id